code/style.css

```diff
@@ -1,4 +1,385 @@
 html {
-  background: #0025ff;
+  background: white;
   color: #fff;
+  font-family: 'Roboto Mono', monospace;
 }
+
+.wrapper{
+  justify-content: center;
+  display: flex;
+flex-direction: column;
+animation: fadeIn 5s;
+opacity: 1;
+}
+@keyframes fadeIn{
+  from{
+    opacity: 0;
+  }
+  to{
+    opacity: 1;
+  }
+  }
+
+header{
+display: flex;
+justify-content: space-evenly;
+align-self: center;
+align-items: center;
+flex-direction: row;
+}
+
+.meny{
+  display: inline;
+}
+.menu-line{
+  width: 35px;
+  height: 5px;
+  background-color: #25838A;
+  margin: 6px 0;
+  border-radius: 10px;
+  align-self: flex-end;
+}
+.nav1:hover, .nav2:hover, .nav3:hover, .nav4:hover{
+text-decoration:underline;
+color: #25838A;
+}
+.logo{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.logo-big{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.big-news{
+  width: 100%;
+  /* border: 2px solid #25838A; */
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  max-width: 500px;
+
+}
+
+.big-news-image{
+  width: 95%;
+  padding: 1rem;
+  border-radius: 20px;
+  object-fit: cover;
+overflow: hidden;
+
+}
+
+.news-card{
+  width: 90%;
+  max-width: 500px;
+  max-height: 300px;
+  border: 2px solid #F2F2F2;
+  /* box-shadow: 5px 5px 5px #25838A; */
+  border-radius: 10px;
+  margin: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  box-shadow:3px 3px 10px #383C4B;
+}
+.news-card:hover{
+  box-shadow: 5px 5px 5px #25838A;
+}
+
+.news-card-image{
+  width: 95%;
+  height: 200px !important;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: 20px;
+  object-fit: cover;
+  overflow: hidden;
+}
+
+h1{
+  color: #383C4B;
+  font-weight:900;
+  font-size: 2rem;
+  align-self: center;
+}
+
+h2{
+  color:#383C4B;
+  font-weight: 400;
+  margin: 0.5rem;
+  font-size: 1.5rem;
+}
+h3{
+  margin: 1rem 1rem;
+  align-self: flex-start;
+  color:#383C4B;
+  font-weight: 400;
+  font-size: 1.5em;
+  text-overflow: ellipsis;
```

Comment:
loved this! my headings were all different length so the cards looked different. will for sure try this out next time!